sylius/resource-bundle/1.9/config/packages/sylius_resource.yaml

```diff
@@ -0,0 +1,15 @@
+# @see https://github.com/Sylius/SyliusResourceBundle/blob/master/docs/index.md
+sylius_resource:
+    # Override default settings
+    #settings:
+
+    # Configure the mapping for your resources
+    mapping:
+        paths:
+            - '%kernel.project_dir%/src/Entity'
```

Comment:
Having the default value here is redundant.

Reply:
Yes this is linked to the removed default value that is not merged yet. So WE Can remove that for now and wait for the further decision and making another PR.